### ISSUE TITLE
NVDA table navigation does not always work on Firefox

### ISSUE THREAD

**Issue by bingen13** [nvaccess/nvda#16511]
### Steps to reproduce:

Unfortunately, I don't have a reliable way to reproduce. It seems to occur when NVDA has been running for a while.

* On Firefox, visit a page with a table.
* Press a cell navigation command (for example, ctrl+alt+right-arrow).

### Actual behavior:

NVDA says edge of table and fails to move to the next cell.

### Expected behavior:

NVDA should navigate to the next cell on the table column and speak it.

### NVDA logs, crash dumps and other attachments:

```
IO - inputCore.InputManager.executeGesture (08:02:26.107) - winInputHook (2088):
Input: kb(desktop):alt+control+rightArrow
IO - speech.speech.speak (08:02:26.130) - MainThread (9040):
Speaking ['Edge of table']
IO - braille.BrailleBuffer.update (08:02:26.131) - MainThread (9040):
Braille regions text: ['Edge of table']
IO - braille.BrailleHandler.update (08:02:26.131) - MainThread (9040):
Braille window dots: 157 145 1245 15 - 135 124 - 2345 1 12 123 15
ERROR - scriptHandler.executeScript (08:02:26.139) - MainThread (9040):
error executing script: <bound method DocumentWithTableNavigation.script_nextColumn of <virtualBuffers.gecko_ia2.Gecko_ia2 object at 0x01379AF0>> with gesture 'alt+ctrl+right arrow'
Traceback (most recent call last):
  File "documentBase.pyc", line 358, in _tableFindNewCell
  File "virtualBuffers\gecko_ia2.pyc", line 592, in _getNearestTableCell
  File "documentBase.pyc", line 285, in _getNearestTableCell
LookupError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "virtualBuffers\gecko_ia2.pyc", line 574, in _getTableCellAt
  File "comtypes\_memberspec.pyc", line 482, in __call__
_ctypes.COMError: (-2147023179, 'Interfaz desconocida.', (None, None, None, 0, None))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "documentBase.pyc", line 384, in _tableFindNewCell
  File "virtualBuffers\gecko_ia2.pyc", line 582, in _getTableCellAt
LookupError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "scriptHandler.pyc", line 295, in executeScript
  File "documentBase.pyc", line 480, in script_nextColumn
  File "documentBase.pyc", line 415, in _tableMovementScriptHelper
  File "documentBase.pyc", line 386, in _tableFindNewCell
RuntimeError: ('Unable to find current cell.', LookupError())
```

#### NVDA installed/portable/running from source:

NVDA installed.

#### NVDA version:

Version: 2024.1 (2024.1.0.31547)

#### Windows version:

Windows 10 22H2 (AMD64) build 19045.4291

#### Name and version of other software in use when reproducing the issue:

Firefox 125.0.3.

#### Does the issue still occur after restarting your computer?

Restarting NVDA itself fixes the issue, but it arises again eventually.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No.

#### If NVDA add-ons are disabled, is your problem still occurring?

Don't know.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes.

**Comment by jmdaweb** [comment]
Same problem here with all add-ons disabled and Windows 11 23H2 (build 22631.3527).

**Comment by Adriani90** [comment]
I can also reproduce this reliably, especially when generating a diff for two files on diffcheck (https://diffcheck.io/). The diff is shown as a table and table navigation fails every time when the diff is generated.
Restarting NVDA makes table navigation work again though.

cc: @jcsteh

**Comment by jcsteh** [comment]
Duplicate of #15862.

**Comment by jcsteh** [comment]
@Adriani90, I just tried on diffcheck and couldn't reproduce it immediately. When you say reliably, I assume you mean you can restart NVDA, then immediately reproduce this on diffcheck with a specific set of files? How big was the diff you produced? Can you provide precise steps to reproduce this reliably? That might help debug this, since I rarely ever see this and am thus not in a position to debug it when it occurs.

**Comment by Adriani90** [comment]
Hmm it seems with NVDA last alpha and Firefox 125.0.3 I cannot reproduce it at all anymore.
